core/src/main/scala/ru/tinkoff/codefest/storage/Storage.scala

```diff
@@ -0,0 +1,8 @@
+package ru.tinkoff.codefest.storage
+
+import scala.language.higherKinds
```

Comment:
scala compiler options already has it

Reply:
I think this line is to please IDEA

Reply:
seems that explicit flag had already pleased IDEA